docs/superpowers/specs/assets/2026-06-03-cli-device-login-authorize-mockup.html

```
@@ -0,0 +1,176 @@
+      <div class="mt-4 flex items-center gap-3 rounded-xl bg-slate-50/80 px-4 py-3">
+        <svg class="h-5 w-5 text-slate-400" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2"><rect x="2" y="3" width="20" height="14" rx="2"/><path d="M8 21h8M12 17v4"/></svg>
+        <div class="min-w-0 text-sm">
+          <p class="font-medium text-slate-700">MacBook-Pro · zsh</p>
+          <p class="text-slate-400">capgo-cli v6.2.0 · requested just now</p>
+        </div>
+      </div>
+
+      <div class="mt-6 space-y-5">
+        <div>
+          <label class="mb-1.5 block text-sm font-medium text-slate-700">Token name</label>
+          <input type="text" value="CLI – MacBook-Pro"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)